AUTHORS.txt

```diff
@@ -0,0 +1,11 @@
+# This is the official list of project authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS.txt file.
+# See the latter for an explanation.
+#
+# Names should be added to this file as:
+# Name or Organization <email address>
+
+Pablo Impallari <impallari@gmail.com>
+Rodrigo Fuenzalida <fuenzarodrigo@gmail.com>
+Dan O. Williams <daniel.williams@gsa.gov>
+USWDS
```

Comment:
I wonder that Dan should go into the CONTRIBUTORS file instead of AUTHORS since he is not a copyright holder (in jurisictions where USWDS is)